src/theme.js

```diff
@@ -0,0 +1,20 @@
+const theme = {
+  primary: '#FF4E54',
+  secondary: '#FF8242',
+  success: '#07CB6E',
+  warning: '#F3B664',
+  cardBorderColor: '#E49393',
```

Comment:
dont need to add color after color name @Arbina001 

also from 13-16 change name of that color